src/styles/animation.scss

```diff
@@ -0,0 +1,21 @@
+@keyframes time {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  25% {
+    transform: rotate(90deg);
+  }
+
+  50% {
+    transform: rotate(180deg);
+  }
+
+  75% {
+    transform: rotate(270deg);
+  }
```

Comment:
why do you need all these options? it would be the same result if you left 0% and 100%